lms/models/course.py

```diff
@@ -5,7 +5,7 @@
 from lms.models.application_settings import ApplicationSettings
 
 
-class Course(BASE):
+class _Course(BASE):
```

Comment:
I renamed as "_Course" and then been importing it `as LegacyCourse`, maybe I should just rename it to that?